Mission involves creep.lua

```diff
@@ -0,0 +1,86 @@
+local argsUtil = require("Module:ArgsUtil")
+local cargoUtil = require("Module:CargoUtil")
+
+local p = {}
+
+-- kill ranges based on difficulty level
+local killRanges = {
+    easy = { 4, 12 },
+    hard = { 2, 5 },
+    special = { 4, 8 }
+}
+
+-- Function to calculate the reward range based on the kill range
+local function calculateRewardRange(lowerBound, upperBound)
+    local lowerReward = 20 * lowerBound * 1.5
+    local upperReward = 20 * upperBound * 1.5
+    return lowerReward .. "-" .. upperReward
+end
+
+function p.main(frame)
+    local args = argsUtil.merge(frame.args, frame:getParent().args)
```

Comment:
util_args.merge() doesn't require any arguments! It'll automatically grab from frame :)

(unless you rewrote from the default)